### PR TITLE
fix labeling

### PR DESCRIPTION
**Pull request by armancohan**
small bug in labeling trec eval file

no effect on results as the label is ignored by trec_eval

### CODE REVIEW

File: scidocs/user_activity_and_citations.py [hunk 1]
```diff
@@ -166,10 +166,10 @@ def make_run_from_embeddings(qrel_file, embeddings, run_file, topk=5, generate_r
             # output is in this format: [qid iter paperid rank similarity run_id]
             if sorted_dists[i][0] in added:
                 continue
-            if i < topk:
-                results.append([pid, '0', sorted_dists[i][0], '1', str(np.round(sorted_dists[i][1], 5)), 'n/a'])
-            else:
+            if i < len(sorted_dists) - topk:
                 results.append([pid, '0', sorted_dists[i][0], '0', str(np.round(sorted_dists[i][1], 5)), 'n/a'])
```
Comment (sergeyf) [inline scidocs/user_activity_and_citations.py:170]: Should this be '1'? Now both are '0'.

Comment (armancohan) [reply scidocs/user_activity_and_citations.py:170]: check line 172. It is already adding '1' for topk results.

Comment (sergeyf) [reply scidocs/user_activity_and_citations.py:170]: Line 172 has this on my screen: `results.append([pid, '0', sorted_dists[i][0], '1', str(np.round(sorted_dists[i][1], 5)), 'n/a'])`

Comment (armancohan) [reply scidocs/user_activity_and_citations.py:170]: that is right (see the 4th column). the format is according to this:
`query-id Q0 document-id rank score STANDARD`

Comment (sergeyf) [reply scidocs/user_activity_and_citations.py:170]: Oh right :)

**Pull request merged**